demo.js

```diff
@@ -317,6 +321,12 @@ function createAlert(html,$container){
 
 	var $alert=$($("template#alert-template").html());
 
+	$alert.find("[alert-type]").html(type);
+
+	if (type == "Error") {
+		$alert.addClass('alert-danger');
```

Comment:
Should we also remove already added 'alert-warning' class here?

Reply:
Oh.. I think this is just for text. Got it. Ignore previous comment.

Reply:
Since danger takes priority over warning, I thought to keep it intact.